docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -93,91 +102,114 @@ const session = new Session({ hostname: "jsonplaceholder.typicode.com"});
 })();
 ```
 
-### Checkpoint one
+### Verify your build with the programmatic API **[correct?]**
+
+In your terminal, issue `npm run build` in your terminal to verify a clean compilation and confirm that no lint errors are present.
 
-Issue `npm run build` to verify a clean compilation and confirm that no lint errors are present. At this point in this tutorial, you have a programmatic API
-that will be used by your handler or another Node.js application. Next you'll define the command syntax for the command that will use your programmatic Node.js APIs.
+At this point, you have a programmatic API to be used by your handler or another Node.js application. Next, define the command syntax for the command that uses your programmatic Node.js APIs.
 
 ### Creating a command definition
 
-Within Zowe CLI, the full command that we want to create is `zowe zowe-cli-sample list typicode-todos`. Navigate to `zowe-cli-sample-plugin/src/cli/list` and create a folder
-`typicode-todos`. Within this folder, create `TypicodeTodos.definition.ts`. Its content should be as follows:
-```typescript
-import { ICommandDefinition } from "@zowe/imperative";
-export const TypicodeTodosDefinition: ICommandDefinition = {
-    name: "typicode-todos",
-    aliases: ["td"],
-    summary: "Lists typicode todos",
-    description: "List typicode REST sample data",
-    type: "command",
-    handler: __dirname + "/TypicodeTodos.handler",
-    options: [
-        {
-            name: "id",
-            description: "The todo to list",
-            type: "number"
-        }
-    ]
-};
+This tutorial creates the following command in Zowe CLI:
+
+```
+zowe zowe-cli-sample list typicode-todos`
 ```
-This describes the syntax of your command.
+
+#### Describing the syntax of your command **[correct?]**
+
+1. Navigate to `zowe-cli-sample-plugin/src/cli/list` and create a folder titled `typicode-todos`. 
+
+2. In this folder, create a file named `TypicodeTodos.definition.ts`.
+
+    Edit the `TypicodeTodos.definition.ts` file to contain the following code:
+
+    ```typescript
+    import { ICommandDefinition } from "@zowe/imperative";
+    export const TypicodeTodosDefinition: ICommandDefinition = {
+        name: "typicode-todos",
+        aliases: ["td"],
+        summary: "Lists typicode todos",
+        description: "List typicode REST sample data",
+        type: "command",
+        handler: __dirname + "/TypicodeTodos.handler",
+        options: [
+            {
+                name: "id",
+                description: "The todo to list",
+                type: "number"
+            }
+        ]
+    };
+    ```
+
+    The `TypicodeTodos.definition.ts` file describes the syntax of your command.
 
 #### Defining command to list group
```

Comment:
I think this section title should be changed to "Adding a command to a command group" if we want to keep it generic.

It would be good to have a sentence introducing what we are showing in this section, e.g.:
"Add the newly created `TypicodeTodosDefinition` to the `list` command group to enable users to list todos by running the `zowe zowe-cli-sample list typicode-todos` command"